stable/kong/values.yaml

```diff
@@ -70,6 +70,11 @@ proxy:
 # Set runMigrations to run Kong migrations
 runMigrations: true
 
+migrations:
```

Comment:
This name is ambiguous.
As a user, when I read it, I feel this is the image used to run migrations for Kong, which is not the case. 

How does a name like `migrationsInitialWaitImage`? That also doesn't sound like a good name.
@shashiranjan84 thoughts?

Reply:
@hbagdi Good point. I agree the name could be better. I'm noticing now that this image is only used if postgres is enabled. So maybe:

```
postgresql:
  waitImage:
    repository: busybox
    tag: latest
```

I was trying to keep the nested `repository` and `tag` keys to match what's done for the kong image in `values.yaml`. I'll defer to what the maintainers think is best.

Reply:
I've made the change in my comment above, but am happy to change if a better solutions exists.

Reply:
@thirstydeveloper `waitImage` seems appropriate at the moment.

Although, it is applicable only to `postgres` at the moment, it will be included for Cassandra in the future so let's skip the `postgresql` parent and keep it generic.

Reply:
👍 Done